### DISCUSSION TITLE
api 文档中，下面几种无序列表文字的格式，是否需要统一？

### DISCUSSION THREAD

**Discussion by WBbug** [antvis/X6#3466] [General]
[NodeAnchor](https://x6.antv.antgroup.com/api/registry/node-anchor)
<img width="1296" alt="image" src="https://user-images.githubusercontent.com/87802698/229545587-e10907e0-dad5-4140-acf4-aab4bb261a2a.png">
[EdgeAnchor](https://x6.antv.antgroup.com/api/registry/edge-anchor)
<img width="1070" alt="image" src="https://user-images.githubusercontent.com/87802698/229545797-870084fa-d632-4a56-a842-0bd6958d25bd.png">
[ConnectionPoint](https://x6.antv.antgroup.com/api/registry/connection-point)
<img width="1249" alt="image" src="https://user-images.githubusercontent.com/87802698/229545884-3a95ab8a-3a5c-4ca1-b42c-c7272ea62b14.png">

需要统一的话，统一成什么样子？
允许的话我这边可以提pr

**Comment by NewByVector** [comment]
非常感谢你对 X6 社区的贡献，我们希望是第二张截图的样式。
